cmd/clusterctl/examples/openstack/README.md

```diff
@@ -18,31 +18,36 @@
 ## Prerequisites
 
 1. Install `yq` (see [here](https://github.com/mikefarah/yq)).
+2. Install `kustomize` v1.0.11 which can be found [here](https://github.com/kubernetes-sigs/kustomize/releases/tag/v1.0.11).
 
 ## Generation
 For convenience, a generation script which populates templates based on openstack cloud provider
 configuration is provided.
 
 1. Run the generation script.
 ```
-./generate-yaml.sh --provider-os [os name]
+./generate-yaml.sh [options] <path/to/clouds.yaml> openstack --provider-os [os name]
```

Comment:
generate-yaml.sh [options] <path/to/clouds.yaml> <cloud> <provider os: [centos,ubuntu,coreos]>


this is current grammar

Reply:
seems we need to provide the os cloud, refer below code:

```
# Check if os cloud is provided
if [[ -n "$2" ]] && [[ $2 != -* ]] && [[ $2 != --* ]]; then
  CLOUD=$2
else
  echo "Error: No cloud specified"
  echo "You mush specify which cloud you want to use."
  echo ""
  print_help
  exit 1
fi
```

Reply:
I will update the [os name] to [[centos,ubuntu,coreos]]